go/README.md

```diff
@@ -1 +1,177 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Tooling
+
+- Visual Studio Code
+- delve
+- golangci-lint
+- staticcheck
+- modd
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
+
+## project structure for an API / Web application
+
+main
```

Comment:
I don't see why start is necessary, start should be the root.

## project structure for an API / Web application
main / start
- business / app (separation of concerns)
  - domain (e.g.: users, cars, chairs etc. etc.)
	- handlers
	- middleware (local middelware)
	- models
    - queries (if you work with a database)
    - repositories
    - services
    - validators
- config
- datamodels
- helpers
- middleware (global middelswares)
- web / server

Reply:
capWeb.Authenticate > capilaMiddleware.Au....

Reply:
config
go > src folder
dist